### ISSUE TITLE
Editor Crash when creating and moving folders

### ISSUE THREAD

**Issue by allenwp** [godotengine/godot#98464]
### Tested versions

Reproducible in:
- v4.4.dev.gh [533c616cb]

### System information

Windows 10.0.22631 - Multi-window, 2 monitors - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 980 Ti (NVIDIA; 32.0.15.6094) - 13th Gen Intel(R) Core(TM) i7-13700K (24 threads)

### Issue description

Creating new sub-folders causes the sub-folder to appear under an mysterious additional `res://` folder until the FileSystem is reloaded.

![image](https://github.com/user-attachments/assets/70621159-57c5-4fb6-80d7-f17f81f624a3)

Moving the new sub-folder as follows, before the FileSystem is reloaded, causes a crash:
![image](https://github.com/user-attachments/assets/436a71dd-a028-4e2c-8418-59c31c783fd9)

```
================================================================
CrashHandlerException: Program crashed
Engine version: Godot Engine v4.4.dev.custom_build (ba513dc5c8f36ab6a145f4b1230e3520d3a4cc00)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[0] CowData<EditorFileSystemDirectory::FileInfo *>::_get_size (C:\Personal\Godot\godot_src\core\templates\cowdata.h:128)
[1] CowData<EditorFileSystemDirectory::FileInfo *>::size (C:\Personal\Godot\godot_src\core\templates\cowdata.h:181)
[2] Vector<EditorFileSystemDirectory::FileInfo *>::size (C:\Personal\Godot\godot_src\core\templates\vector.h:94)
[3] EditorFileSystemDirectory::get_file_count (C:\Personal\Godot\godot_src\editor\editor_file_system.cpp:89)
[4] FileSystemDock::_before_move (C:\Personal\Godot\godot_src\editor\filesystem_dock.cpp:2050)
[5] FileSystemDock::_move_operation_confirm (C:\Personal\Godot\godot_src\editor\filesystem_dock.cpp:2006)
[6] FileSystemDock::drop_data_fw (C:\Personal\Godot\godot_src\editor\filesystem_dock.cpp:3062)
[7] call_with_variant_args_helper<FileSystemDock,Vector2 const &,Variant const &,Control *,0,1,2> (C:\Personal\Godot\godot_src\core\variant\binder_common.h:304)
[8] call_with_variant_args<FileSystemDock,Vector2 const &,Variant const &,Control *> (C:\Personal\Godot\godot_src\core\variant\binder_common.h:419)
[9] CallableCustomMethodPointer<FileSystemDock,void,Vector2 const &,Variant const &,Control *>::call (C:\Personal\Godot\godot_src\core\object\callable_method_pointer.h:111)
[10] Callable::callp (C:\Personal\Godot\godot_src\core\variant\callable.cpp:58)
[11] CallableCustomBind::call (C:\Personal\Godot\godot_src\core\variant\callable_bind.cpp:153)
[12] Callable::callp (C:\Personal\Godot\godot_src\core\variant\callable.cpp:58)
[13] Control::drop_data (C:\Personal\Godot\godot_src\scene\gui\control.cpp:1968)
[14] Viewport::_gui_drop (C:\Personal\Godot\godot_src\scene\main\viewport.cpp:1713)
[15] Viewport::_perform_drop (C:\Personal\Godot\godot_src\scene\main\viewport.cpp:2191)
[16] Viewport::_gui_input_event (C:\Personal\Godot\godot_src\scene\main\viewport.cpp:1816)
[17] Viewport::push_input (C:\Personal\Godot\godot_src\scene\main\viewport.cpp:3177)
[18] Window::_window_input (C:\Personal\Godot\godot_src\scene\main\window.cpp:1682)
[19] call_with_variant_args_helper<Window,Ref<InputEvent> const &,0> (C:\Personal\Godot\godot_src\core\variant\binder_common.h:304)
[20] call_with_variant_args<Window,Ref<InputEvent> const &> (C:\Personal\Godot\godot_src\core\variant\binder_common.h:419)
[21] CallableCustomMethodPointer<Window,void,Ref<InputEvent> const &>::call (C:\Personal\Godot\godot_src\core\object\callable_method_pointer.h:111)
[22] Callable::callp (C:\Personal\Godot\godot_src\core\variant\callable.cpp:58)
[23] Callable::call<Ref<InputEvent> > (C:\Personal\Godot\godot_src\core\variant\variant.h:906)
[24] DisplayServerWindows::_dispatch_input_event (C:\Personal\Godot\godot_src\platform\windows\display_server_windows.cpp:3772)
[25] DisplayServerWindows::_dispatch_input_events (C:\Personal\Godot\godot_src\platform\windows\display_server_windows.cpp:3743)
[26] Input::_parse_input_event_impl (C:\Personal\Godot\godot_src\core\input\input.cpp:803)
[27] Input::flush_buffered_events (C:\Personal\Godot\godot_src\core\input\input.cpp:1084)
[28] DisplayServerWindows::process_events (C:\Personal\Godot\godot_src\platform\windows\display_server_windows.cpp:3225)
[29] OS_Windows::run (C:\Personal\Godot\godot_src\platform\windows\os_windows.cpp:1772)
[30] widechar_main (C:\Personal\Godot\godot_src\platform\windows\godot_windows.cpp:181)
[31] _main (C:\Personal\Godot\godot_src\platform\windows\godot_windows.cpp:206)
[32] main (C:\Personal\Godot\godot_src\platform\windows\godot_windows.cpp:220)
[33] WinMain (C:\Personal\Godot\godot_src\platform\windows\godot_windows.cpp:234)
[34] __scrt_common_main_seh (D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288)
[35] <couldn't map PC to fn name>
-- END OF BACKTRACE --
================================================================
```

### Steps to reproduce

1. Create a new project
2. Create a new folder named `a` in `res://`
3. Create a new folder named `b` in `a`
4. Notice that `b` appears as `res://a/res://b`
5. Move `b` into `res://`
6. Crash

### Minimal reproduction project (MRP)

N/A

**Comment by VetusChronos** [comment]
I tried to reproduce this here on my Linux system (Manjaro KDE) and the filesystem worked correctly.

This bug probably only occurs on Windows systems

Reproduce in: v4.4.dev.custom_build [533c616cb]

**Comment by matheusmdx** [comment]
This is a duplicate of #98045 and will be fixed by #98075

**Comment by allenwp** [comment]
Closing as duplicate of #98045
